odoo/addons/base/data/ir_module_module.xml

```diff
@@ -44,6 +44,20 @@
             <field name="website">https://www.odoo.com/app/accounting?utm_source=db&amp;utm_medium=module</field>
         </record>
 
+        <record model="ir.module.module" id="base.module_knowledge">
+            <field name="name">knowledge</field>
+            <field name="sequence">30</field>
+            <field name="shortdesc">Knowledge</field>
+            <field name="category_id" ref="base.module_category_productivity_knowledge"/>
```

Comment:
Could you use `"base.module_category_productivity"` instead (and remove the `module_category_productivity_knowledge` record) ?

Reply:
hi @abd-msyukyu-odoo , thanks for the feedback i have updated the category as suggested. but in the knowledge module manifest file the category is given as Productivity/Knowledge, so should we need to change here also ?

looking for your thoughts

Reply:
Hello again! No need to change the Enterprise category for Knowledge.

Indeed, in Enterprise the category is different, and its purpose is to contain future optional modules for Knowledge. But in Odoo Community, the `ir.module.module` record only redirects to the Odoo website (to propose the upgrade plans, or to more information about the module), so we don't need the specific category. That record is not used when using Odoo Enterprise.

Reply:
@abd-msyukyu-odoo thanks for the clarification :)